MoTrPAC/motrpac-frontend
# Connectes to #137. Team page update.

### Key Changes:
* Added new members and updated some existing members' roles on the portal's team page.

### Technical note:
* Make sure you `node.js` version is at `v14.15.4` or above.
* Create a`.env` file at the repo root with `ESLINT_NO_DEV_ERRORS=true` in it.

### Steps to test:
1. Go to `^/team` page and expect to see members such as "Mihir Samdarshi" and "Samiya Shimly" on the page.
